[EDU-3901]  create service (#650)
aziontech/docs: src/content/docs/en/pages/deploy-journey/create-service/create-service.mdx

---
title: Create an edge service
description: aaa
meta_tags: deploy, orchestrator
namespace: docs_deploy_overview
menu_namespace: deployMenu
og_image: ''
permalink: /documentation/products/deploy/create-edge-service/
---

You can create an edge service by using:

- [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/)

> - [Azion API](https://api.azion.com/)

Reviewer: alt: levar para H2 e incluir os links específicos em cada seção.

---

## Via RTM 

To create an edge service, proceed as follows:

1. Go to [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Libraries** > **Edge Services**.
2. Click on the **Add Service** button.
- Your service will be created automatically.

> - You can change the name of the service by clicking on **My New Service** in the name bar.

Reviewer 2: name field?

Author: ```suggestion
- You can change the name of the service by clicking on **My New Service** in the name field.
```

3. Go to the **Resources** tab and click **Add Resource**.

> 4. Configure the resources you need for your service, using the triggers: **Install**, **Reload**, and **Uninstall**.

Reviewer 2: and ou or?

Author: ```suggestion
4. Configure the resources you need for your service, using the triggers: **Install**, **Reload**, or **Uninstall**.
```

5. **Optional**: where you've used variables in the content of one or more resources, you can set the default values for them in the **Environment** tab.
- The variables must be in the format `Variable = Value`, where `Variable` has been used in the content of resources that are already registered.

> 6. Activate the service by enabling the switch in the left-bottom corner in the **Environment** tab.

Reviewer 2: ```suggestion
6. Activate the service by enabling the switch in the bottom-left corner in the **Environment** tab.
```

Reviewer 2: para padronizar com as demais docs do portal

> 1. Go to [Real-Time Manager (RTM)](https://manager.azion.com/).
> 2. On the top-left corner, select the **Products menu**, represented by three horizontal lines, and then **Edge Libraries** > **Edge Services**.
> 3. Click on the **Add Service** button.
> > Your service will be created automatically. You can change the name of the service by clicking on **My New Service** in the name bar.
> 4. Go to the **Resources** tab and click **Add Resource**.
> 5. Configure the resources you need for your service, using the triggers: "*Install*", "*Reload*", and "*Uninstall*".
> 6. **Optional**: where you've used variables in the content of one or more resources, you can set the default values for them in the **Environment** tab.
> > The variables must be in the format `Variable = Value`, where `Variable` has been used in the content of resources that are already registered.
> 7. Activate the service and click on the **Save** button.

Reviewer: ```suggestion
1. Go to [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Libraries** > **Edge Services**.
2. Click on the **Add Service** button.
- Your service will be created automatically.
- You can change the name of the service by clicking on **My New Service** in the name bar.
3. Go to the **Resources** tab and click **Add Resource**.
4. Configure the resources you need for your service, using the triggers: **Install**, **Reload**, and **Uninstall**.
5. **Optional**: where you've used variables in the content of one or more resources, you can set the default values for them in the **Environment** tab.
- The variables must be in the format `Variable = Value`, where `Variable` has been used in the content of resources that are already registered.
6. Activate the service by enabling the switch in the left-bottom corner in the **Environment** tab.
7. Click on the **Save** button.
```

--- 

## Via API

1. Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and informing the name of the service being created:

```bash
curl --location 'https://api.azionapi.net/edge_services/' \
--header 'Accept: application/json; version=3' \
--header 'Authorization: Token [TOKEN VALUE]' \
--header 'Content-Type: application/json' \
--data '{
  "name": "string"
}'
``` 

Now, the service is available and able to be deployed onto the edge nodes.

> 

Reviewer: ```suggestion
```
